docs/topics/exceptions.md

```diff
@@ -1,122 +1,679 @@
 [//]: # (title: Exceptions)
 
-## Exception classes
+Exceptions are key tools for handling runtime errors. They are designed to keep the program running predictably despite encountering 
+issues that could disrupt its normal operation.
+Kotlin simplifies exception management by treating all exceptions as unchecked. 
+This means that exceptions can be caught but are not required to be explicitly handled or [declared](https://kotlinlang.org/docs/java-to-kotlin-interop.html#checked-exceptions), unlike in Java. 
+
+For more information about how Kotlin handles exceptions when interacting with Java, Swift, and Objective-C, see 
+[Exception interoperability with Java, Swift, and Objective-C](#exception-interoperability-with-java-swift-and-objective-c).
+
+Exceptions are represented by subclasses of the 
+[`Exception`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-exception/) class, which are subclasses of the 
+[`Throwable`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-throwable/) class. For more information about the 
+hierarchy, see the [Exception hierarchy](#exception-hierarchy) section. Since `Exception` is an open 
+class, you can create [custom exceptions](#creating-custom-exceptions) to suit your application's specific needs.
+
+Working with exceptions consists of two primary actions:
+
+* **Throwing exceptions:** Indicate the occurrence of a problematic situation.
+* **Catching exceptions:** Manage unexpected scenarios by remedying the issue or notifying the user.
+
+## Throwing exceptions
+
+All exception classes in Kotlin inherit from the [`Throwable`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-throwable/) class,
```

Comment:
hmm 🤔 - it's there to explain where the `throw` keyword is coming from - but how about this version instead?

You can manually throw exceptions using the `throw` keyword because all exception classes inherit from the  [`Throwable`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-throwable/) class.
Throwing an exception indicates that an unexpected runtime error has occurred in the code. In Kotlin, exceptions are [objects](classes.md#creating-instances-of-classes).
When you throw an exception, an instance of an exception class is created.

Reply:
Further refined these sentences into:

You can manually throw exceptions with the `throw` keyword.
Throwing an exception indicates that an unexpected runtime error has occurred in the code.
Exceptions are [objects](classes.md#creating-instances-of-classes), and throwing one creates an instance of an exception class.